server/base/src/main/java/org/apache/accumulo/server/AccumuloDataVersion.java

```diff
@@ -47,18 +47,6 @@ public class AccumuloDataVersion {
    */
   public static final int ROOT_TABLET_META_CHANGES = 10;
 
-  /**
-   * version (9) reflects changes to crypto that resulted in RFiles and WALs being serialized
-   * differently in version 2.0.0. Also RFiles in 2.0.0 may have summary data.
-   */
-  public static final int CRYPTO_CHANGES = 9;
-
-  /**
-   * version (8) reflects changes to RFile index (ACCUMULO-1124) AND the change to WAL tracking in
-   * ZK in version 1.8.0
-   */
-  public static final int SHORTEN_RFILE_KEYS = 8;
-
```

Comment:
The javadocs should be moved to the "historic data versions" section below.

Reply:
Addressed in a58a977761,